openid-4-verifiable-presentations-1_0.md

```diff
@@ -659,7 +659,7 @@ The error response follows the rules as defined in [@!RFC6749], with the followi
 
 `invalid_request`:
 
-- The request contains more than one out of the following three options to communicate a requested Credential: a `presentation_definition` parameter, a `presentation_definition_uri` parameter, or a scope value representing a Presentation Definition.
+- The request contains none or more than one out of the following three options to communicate a requested Credential: a `presentation_definition` parameter, a `presentation_definition_uri` parameter, or a scope value representing a Presentation Definition.
```

Comment:
We may want to make it clearer that a wallet isn't *required* to return an error in the 'none of' case (but is required to return an error in the "more than one" case). [As the wallet may support other protocols, e.g. SIOPv2]

Reply:
In the context of OpenID4VP it is required to provide one of the options, otherwise it is not an OpenID4VP request. Especially, I would expect an error if response_type is vp_token. I agree that for response_type code, the situation is different. If the request is also a SIOPv2 request using vp_token id_token, then I would still expect an error.

Reply:
Perhaps we should think about a dedicated scope value that clearly defines that the request is an OpenID4VP request?

Reply:
I updated the language in the PR. Would you agree with the new language @jogu ?